### DISCUSSION TITLE
2팀 최종 팀회고

### DISCUSSION THREAD

**Discussion by sung-eunji** [codeit-fe2-2/taskify#120] [Q&A]
# keep

# Problem

# Try

**Comment by ckswnskfk** [comment]
# Keep
- 브랜치 전략
- github을 활용한 문서화 -> 여러 군데에 왔다갔다 하지 않아도 돼서 보기 편함

# Problem
- 코드 리뷰 할 에너지 부족
- 너무 널널했던 컨벤션(코드, 폴더 구조, 파일 등)
- 기술 스택 선정 근거 부족
- 요구 사항 분석 미흡

# Try
- 프로젝트 시작 전 전체 요구 사항 분석과 코드 컨벤션을 정하는 데 많은 시간과 리소스를 쓰자
- 코드 리뷰 스케쥴을 정하는 건 어떨까?

**Comment by BBoBBoM** [comment]
# keep
- 깃헙 프로젝트 관리에 대한 To do ~done 등에 관한 작업단위의 정함이 프로젝트 진행 방향, 진도,  앞으로 해야할 일에 대한 부분 파악하기가 쉬웠습니다
- 팀원간 소통하거나 의견에 대한 조언을 구하거나, 문제해결을 위한 공유 라이브 쉐어에 대한 경험이 좋았습니다. 
# Problem
- pr리뷰에 대한 활발한 참여를 못한 부분이 죄송했습니다. 
- 결국 다른부분에서 공통컴포넌트등 다른부분을 써야하는데 다시 살펴보고 이해해야 했던 부분으로 시간을 허비하는 경우도 있었고 리뷰에 적극 참여하여 미연에 이후 사용할 부분에 적용을 미리 생각해서 이후 사용이나 개선점을 제시하여 작업진행을 좀 더 부드럽게 하는게 좋았을거 같은 아쉬움이 있습니다. 
- 파일등을 작성하거나 사용하는 과정에서 코드는 만들었으나 이것의 용도에 대한 이해 불충분으로 용도에 맞지않는 폴더에 넣거나 한 부분이 포로젝트 폴더구조에 혼란을 주었던거 같습니다. 
# Try
- 모르는 부분이나 다른 코드를 참고할때 적용에만 중점을 두지말고 뜯어보고 원리에 대한 이해 시간을 가지도록 노력하겠습니다
- 마감 기한에대한 좀더 분명한 가이드라인이 있는것은 어떠신지요

**Comment by 00TaciTa00** [comment]
# keep
- 우리 깃허브 사용 진짜 잘 한 거 같죠?
- 우리 업무 분담 꽤 잘 한 거 같죠?

# Problem
- 적정 마감 기한을 못 지킨 거 같아요. 앞으론 마감기한을 미리 정해두거나 하는 식으로 시간 관리를 하는 게 좋을 거 같아요.
- 너무 오래 고민하는...게 있었어요. 고민이 한 시간이 넘는다던가 하면 바로 팀원 등에 물어보는 걸 정해두는 게 좋을 거 같아요.

# Try
- 좀 더 다양한 라이브러리를 찾아보고 싶네요
- 코드 리뷰를 좀 더 꼼꼼히 해보고 싶네요(CLI 써보고 싶어요)
- Tailwind 말고 모듈이나 scss를 써보고 싶네요

**Comment by caseV12** [comment]
# keep
- 깃허브 프로그레스 관리가 좋았습니다.
# Problem
- 마감기한이 필요한 주범입니다... 자꾸 욕심내다가 프로젝트에 기여를 못한 것 같아 죄송스러웠습니다.
- 코드 리뷰 받기만 하고 제가 한 게 몇 개 안되서 이거 또한 반성...
# Try
